README-dockerhub.md

```diff
@@ -50,38 +50,40 @@ Trivy will attempt to resolve the following details from the available environme
+| OVERRIDE_REPOSITORY        | Use this environment variable to explicitly specify the repository name used by Trivy                                                                                |
+| FALLBACK_REPOSITORY        | Use this environment variable as a backup if no other repository env vars can be found                                                                               |
+| OVERRIDE_BRANCH            | Use this environment variable to explicitly specify the branch used by Trivy                                                                                         |
+| FALLBACK_BRANCH            | Use this environment variable as a backup if no other branch env vars can be found                                                                                   |
+| OVERRIDE_BUILDSYSTEM       | Use this environment variable to explicitly specify the build system                                                                                                 |
+| IGNORE_PANIC               | Use this environment variable to return exit code 0 on cli panic error                                                                                               |
+| OVERRIDE_REPOSITORY_URL    | Use this environment variable to explicitly specify the repository link used by Trivy (For result's web link)                                                        |
+| OVERRIDE_REPOSITORY_SOURCE | Use this environment variable to explicitly specify the repository source used by Trivy                                                                              |
+| HTTP_PROXY/HTTPS_PROXY     | Use these environment variable for proxy configuration                                                                                                               |
+| XDG_DATA_HOME              | use this environment variable to designate the base directory for storing user-specific data                                                                         |
+| XDG_CACHE_HOME             | use this environment variable for setting the cache directory                                                                                                        |
+| SAST_LOGS                  | use this environment variable with true value for writing sast logs to a file (The file name is: ${REPOSITORY_NAME}-sast-logs.txt under the SAST_LOGS_DIR directory) |
+| SAST_LOGS_DIR              | use this environment variable to explicitly specify the location where the log file should be written (Default is /tmp/.trivy/plugins/aqua)                          |
+| TRIVY_QUIET                | Disable trivy output report in log                                                                                                                                   |
+| AQUA_ASSURANCE_EXPORT      | The path to export policies results (JSON)                                                                                                                           |
```

Comment:
still missing flags
TRIVY_SKIP_POLICIES - skip assurance policies check

TRIVY_SKIP_RESULT_UPLOAD - disable uploading scan results to aqua platform

TRIVY_SKIP_POLICY_EXIT_CODE - Prevent non-zero exit code if an assurance policy failed

SKIP_PIPELINES - Skip pipeline misconfiguration scan

AQUA_ASSURANCE_EXPORT - export assurance policy result to json

TRIVY_FETCH_DEPTH - configure how may historical commits to fetch for enriching scan results (Default 100, can be disabled passing 0 for repositories with large commit history data)

Reply:
I've added them to the Command Line Arguments section as well (decided not to include the TRIVY_FETCH_DEPTH, it is something internal)

Reply:
ok, but some client with large repositories might use it to mitigate slow scans.
by default it fetches 100 commits.